pkg/puppet/puppet.go

```diff
@@ -391,6 +397,13 @@ func (p *Puppet) writeHieraData(puppetPath string, cluster interfaces.Cluster) e
 			variables = append(variables, `tarmak::etcd_mount_unit: "var-lib-etcd.mount"`)
 		}
 
+		size := strings.Split(strings.ToLower(instancePool.Config().Size), ".")
+		if b, ok := nvmeInstances[size[0]]; ok && b {
+			variables = append(variables, `site_module::ebs_device: "/dev/nvme1n1"`)
```

Comment:
This can be `site_module::docker_storage::ebs_device` instead of `site_module::ebs_device`

Reply:
:+1: